<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomItem</class>
 <widget class="QWidget" name="RoomItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>80</height>
      </size>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>300</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>70</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lb_roomname">
         <property name="text">
          <string>房间名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_roomid">
         <property name="text">
          <string>房间号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_roomcreator">
         <property name="text">
          <string>房主</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
